### ISSUE TITLE
Clone API is not working as expected in ES 7.1

### ISSUE THREAD

**Issue by alexis-anzieu-ssense** [elastic/elasticsearch#58568]
Running on **Elasticsearch 7.1**
Regarding the documentation I should be able to clone an existing index with settings: https://www.elastic.co/guide/en/elasticsearch/reference/7.x/indices-clone-index.html

However when I do this call:
```
PUT index1/_clone/index2
  "settings": {
    "index.number_of_shards": 3,
    "index.number_of_replicas": 0,
    "index.auto_expand_replicas": false,
    "index.routing.allocation.require._name": null,
    "index.blocks.read_only": false
  }
}
```

It sends me back
```
{
  "error": {
    "root_cause": [
      {
        "type": "invalid_type_name_exception",
        "reason": "Document mapping type name can't start with '_', found: [_clone]"
      }
    ],
    "type": "invalid_type_name_exception",
    "reason": "Document mapping type name can't start with '_', found: [_clone]"
  },
  "status": 400
}
```

It seems like a new feature on ES 7 so I should maybe use reindex API instead, or it may be fixed in the newest version, just to let you know. Thank you!

**Comment by jimczi** [comment]
This feature was released in `7.4` so it's not available in the version you're using (`7.1`). 
You should upgrade to a more recent version.

**Comment by alexis-anzieu-ssense** [comment]
Looks legit, thank you!
